### PR TITLE
Reorganize project directories

### PR DESCRIPTION
**Pull request by bitzoic**
## Type of change

- Improvement (refactoring, restructuring repository, cleaning tech debt, ...)

## Changes

The following changes have been made:

Changes the project structure for all projects to:
```
My_app
├── src/
│   └────── main.tsx
├── app-contract
│   ├────── src/main.sw
│   └────── tests/harness.rs
├── package.json
├── index.html
├── README.md
└── SPECIFICATION.md
```

This should allow for quicker access to the project's source code and not be buried in a million directories. 

## Notes

- Some projects such as TicTacToe/NFT have been removed from "games"/"native-assets" and moved into the parent directory
- This may break any other open pull requests that depend on this  project structure
- Empty UI folders have been removed

### CODE REVIEW

**Review by matt-user — CHANGES_REQUESTED** [review]
This broke the vercel deployment.  I'll request/push changes to fix this.

File: TicTacToe/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[workspace]
```
Comment (matt-user) [inline TicTacToe/Cargo.toml:1]: Do we need this file?  seems like all rust stuff should be contained in the contract folder

File: TicTacToe/Forc.toml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+[workspace]
+members = ["./tictactoe-contract"]
```
Comment (matt-user) [inline TicTacToe/Forc.toml:2]: Do we need this file?

File: TicTacToe/Cargo.toml (hunk 1 above)
Comment (bitzoic) [reply TicTacToe/Cargo.toml:1]: These are included so CI can run `forc b` and `cargo t` in the root of the project's folder without special naming for each project's contracts. You can see this here: https://github.com/FuelLabs/sway-applications/blob/39d19079f8b4a4b147337eed33b2a9e8951023f2/.github/workflows/ci.yml#L43

File: TicTacToe/Forc.toml (hunk 2 above)
Comment (bitzoic) [reply TicTacToe/Forc.toml:2]: Same as the comment above. While this project just has one contract, some have more such as the AMM.

**Review by K1-R1 — APPROVED** [review]

**Review by matt-user — APPROVED** [review]

**Pull request merged**